components/SpotlightBanner/SpotlightBanner.tsx

```diff
@@ -1,17 +1,34 @@
+import classNames from 'classnames';
 import React, { Children, FunctionComponent, isValidElement } from 'react';
 import { SpotlightBannerAside } from './SpotlightBannerAside';
 import { SpotlightBannerMain } from './SpotlightBannerMain';
+import { Button } from '../Button';
 
-const SpotlightBanner: FunctionComponent = ({ children }) => {
+interface SpotlightBannerProps {
+  header?: boolean;
```

Comment:
Just add a `className` prop ... any extra classes can be added through that.

Reply:
Replaced `header` with a `className` prop